src/content/en/2021/jamstack.md

```diff
@@ -151,13 +158,13 @@ Trend over countries applies for regions, that's interesting as well.
   sql_file="adoption_by_region.sql"
 ) }}
 
-The number of sites in Europe for desktop is 23K versus mobile 26K, that's 1.1% of all web sites in that region. In the Americas, there're 26K desktop sites versus 24K mobiles sites, 1.2% of sites. In any case, Europe has tiny bit lower adoption of Jamstack than the Americas. Asia has almost the same numbers, 21K desktop and 22K mobile as leading regions but greater Jamstack adoption, 1.45%. We saw that overall site adoption is 1.1%, but looks like Asia is going further with it and drives trends there.
+The number of sites in Europe for desktop is 23k versus mobile 26k, which is 1.1% of all web sites in that region. In the Americas, there are 26k desktop sites and 24k mobiles sites (1.2% of sites). Asia has almost the same numbers with 21k desktop and 22k mobile as the leading region with greater Jamstack adoption at 1.45%. Oceania and Africa have way lower overall numbers, but they have way greater Jamstack adoption. Oceania 2.19% and Africa 2%. Overall site adoption is at 1.1%.
 
-Oceania and Africa have way lower overall numbers, but they have way greater Jamstack adoption. Oceania 2.19% and Africa 2%.
+{# TODO - do you need to list the numbers for each, or should percentages be enough? or numbers for just the first as a baseline #}
```

Comment:
We generally try to avoid absolute numbers and prefer percentages. The make no sense without context of how many they are out of.

However sometimes they can be useful to set context - particularly for small numbers (e.g. "it's only 0.1% but in the context of the web that's still a massive number of 600,000 sites"). But that should be the exception and not over used.